Nesting brackets inside block comments and strings (#93)
Roblox/creator-docs: content/en-us/luau/strings.md

--> Hello 'world'!
```

> If necessary, you can nest multiple brackets inside a string using the same number of equal signs in both the beginning and ending bracket:

Reviewer: ```suggestion
If necessary, you can nest multiple brackets inside a string using the same number of equal signs in both the beginning and ending bracket:

```

--> Hello 'world'!
```

> If necessary, you can nest multiple brackets inside a string using the same quantity of equals sign in both the beginning and the ending bracket.

Reviewer: ```suggestion
If necessary, you can nest multiple brackets inside a string using the same number of equal signs in both the beginning and ending bracket:
```

> 
> The example below uses one equals sign (can be more than one)
> 

Reviewer: ```suggestion
```
